charts/datarepo/ci/testing-values.yaml

```diff
@@ -56,14 +66,16 @@ datarepo-api:
   existingServiceAccountSecretKey: "datareposerviceaccount"
   existingSecretAzure: "test-azure"
   existingApplicationSecretSecretKey: "applicationsecret"
+  existingSynapseUserSecretKey: "test-synapse-user"
```

Comment:
Doesn't this need to match the kubeSecretKey from above (e.g. line 19)?

Reply:
Yes, I think it should match now!